dappnode_package.json

```diff
@@ -35,10 +35,22 @@
   },
   "exposable": [
     {
-      "name": "Erigon Gnosis Chain JSON RPC",
-      "description": "JSON RPC endpoint for Erigon Gnosis Chain (HTTPS & WSS)",
+      "name": "Erigon JSON RPC",
+      "description": "JSON RPC endpoint for Erigon mainnet",
       "serviceName": "erigon",
       "port": 8545
+    },
+    {
+      "name": "Erigon JSON RPC (WS)",
+      "description": "JSON RPC WebSocket endpoint for Erigon mainnet",
+      "serviceName": "erigon",
+      "port": 8545
+    },
+    {
+      "name": "Torrent sync port",
+      "description": "Erigon torrent port for sync",
+      "serviceName": "erigon",
+      "port": 43084
     }
```

Comment:
Now I'm afraid you took it a bit too far. In my understanding this should only contain the HTTP(S) ports that go through the standard reverse proxy. This wouldn't work with a torrent port. I might be wrong though, can you check?

In general I'd say you should stick to the mainnet one so we keep compatibility with it. If we need other changes we'll upstream them before incorporating them here.

Reply:
There are no much info about exposing ports such a way in manifest docs. Agree on sticking to eth package, made it with a flag (but made it configurable env var)